docs/src/less/dox-chrome.less

```diff
@@ -3,10 +3,10 @@
 .ts-doxchrome {
 	#chrome {
 		.ts-mixin-maximize();
-		top: 30px;
+		top: 40px;
```

Comment:
Can we use @ts-unit-double?

Reply:
At this moment, no we can't. It needs to be 40px, if we did double it'll be 44px. We could switch it when we release v11 though.